scripts/build.sh

```diff
@@ -19,4 +19,9 @@ export CGO_ENABLED=0
 
 # Build binary
 cd $GOPATH/src/github.com/raintank/metrictank
-go build -ldflags "-X main.GitHash=$GITVERSION" -o $BUILDDIR/metrictank
+if [ "$1" == "race" ]
+then
+  CGO_ENABLED=1 go build -race -ldflags "-X main.GitHash=$GITVERSION" -o $BUILDDIR/metrictank.race
```

Comment:
i mainly just ran it directly in my terminal. But yeah, you're right i should probably just give it the same name so it can be used to same way. Then i should maybe also make it check for some variable like `$BUILD_RACE` instead of a CLI argument, that way we can use other things like `scripts/launch.sh` with a race build if we just first  set & export `$BUILD_RACE`